engine/engine.go

```diff
@@ -91,12 +91,10 @@ func (r *ruleEngine) RunRules(ctx context.Context, rules []Rule) {
 			case response := <-ret:
 				if !response.conditionResponse.Passed {
 					responses = append(responses, response)
-
 				} else {
 					// Log that rule did not pass
 					fmt.Printf("Rule did not error")
 				}
-				responses = append(responses, response)
```

Comment:
I assume we don't actually want to do this

Reply:
ie we should only return violations and not every result

Reply:
I think so, but I also think that at like a debug level low enough I could see us logging these